Version ~0.2.0 npm compatability [MAR 2021]

I see that `vue-native-router":"~0.2.0”` still requires old packages like

- `“vue-native-core": "~0.1.0”`,
- `"vue-native-helper": "~0.1.0”`,
- `"react-native@":"^0.59.0"`

When install with `npm i`, here are some parts of error

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project@0.2.0
npm ERR! Found: vue-native-core@0.2.0
npm ERR! node_modules/vue-native-core
npm ERR!   vue-native-core@"~0.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vue-native-core@"~0.1.0" from vue-native-router@0.2.0
npm ERR! node_modules/vue-native-router
npm ERR!   vue-native-router@"~0.2.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/AT/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/AT/.npm/_logs/2021-03-28T08_31_00_269Z-debug.log
```
I believe `npm` wasn’t update but please verify if there were some issues left

Thanks in advance

--- a/package.json
+++ b/package.json
@@ -32,9 +32,9 @@
     "react-navigation-tabs": "^2.7.0"
   },
   "peerDependencies": {
-    "vue-native-core": "~0.1.0",
-    "vue-native-helper": "~0.1.0",
-    "vue-native-scripts": "~0.1.0",
+    "vue-native-core": "^0.1.0",
+    "vue-native-helper": "^0.1.0",
+    "vue-native-scripts": "^0.1.0",
     "react-native-safe-area-context": "^3.1.8",
     "react-native-screens": "^2.11.0",
     "react-native-vector-icons": "^7.1.0",

